### PR TITLE
disable building examples on old Mac OSX version (no OpenGL3) and fix for https://github.com/bulletphysics/bullet3/issues/372

### PR DESCRIPTION
**Pull request by erwincoumans**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -187,7 +187,7 @@ ENDIF (OPENGL_FOUND)
 
 
 OPTION(BUILD_BULLET2_DEMOS "Set when you want to build the Bullet 2 demos" ON)
-IF(BUILD_BULLET2_DEMOS)
+IF(BUILD_BULLET2_DEMOS AND BUILD_OPENGL3_DEMOS)
```
Comment (scpeters) [inline CMakeLists.txt]: I think `BUILD_OPENGL3_DEMOS` isn't set until later (lines 205-219)

**Comment by erwincoumans** [comment]
Hi Steven, can you try if this works for you? It should build the ExampleBrowser on older Mac OSX versions, with just limited OpenGL2 enabled (not all examples will show graphics because it requires GL3, but quite a few do).

**Comment by scpeters** [comment]
Looks like [appveyor failed](https://ci.appveyor.com/project/erwincoumans/bullet3/build/1.0.212#L2339). I'll test the compilation on 10.8 with my virtual machine, but I'm not sure OpenGL will work; I'll give it a shot though.

**Comment by scpeters** [comment]
Works great on 10.8!

**Pull request merged**